in/command.go

```diff
@@ -122,14 +122,27 @@ func (command *Command) Run(destinationDir string, request Request) (Response, e
 				return Response{}, err
 			}
 
+			if request.Params.UnpackInto != "" {
+				request.Params.Unpack = true
```

Comment:
I feel like `unpack_into` implies `unpack`. Requiring users to set `unpack: true` when they use `unpack_into` is cumbersome and redundant.